deployment/gce/lily/worker_config.toml

```diff
@@ -0,0 +1,78 @@
+[API]
+  ListenAddress = "/ip4/127.0.0.1/tcp/1234/http"
+  RemoteListenAddress = ""
+  Timeout = "30s"
+
+[Backup]
+  DisableMetadataLog = false
+
+[Logging]
+
+[Libp2p]
+  ListenAddresses = ["/ip4/0.0.0.0/tcp/0", "/ip6/::/tcp/0"]
+  AnnounceAddresses = []
+  NoAnnounceAddresses = []
+  DisableNatPortMap = false
+  ConnMgrLow = 150
+  ConnMgrHigh = 180
+  ConnMgrGrace = "20s"
+
+[Pubsub]
+  Bootstrapper = false
+  RemoteTracer = "/dns4/pubsub-tracer.filecoin.io/tcp/4001/p2p/QmTd6UvR47vUidRNZ1ZKXHrAFhqTJAD27rKL9XYghEKgKX"
```

Comment:
This should be configured in our instance either via an environment variable or adding directly on the config.toml and not part of the general deployment definition.

Reply:
It's already the default? https://github.com/filecoin-project/lily/blob/master/config/config.go#L178

Reply:
I think it should also be removed out of that config. It seems like a forgotten hard-coded value.